src/functions/function-command/invoke-function.ts

```diff
@@ -0,0 +1,397 @@
+        (data.invokeDataMode === 'File'
+          ? data.invokeDataFile !== undefined && data.invokeDataFile?.trim()?.length !== 0
+          : data.invokeDataText !== undefined && data.invokeDataText?.trim()?.length !== 0) &&
+        (data.invokeInstance === 'Local'
+          ? data.invokePath !== undefined && data.invokePath?.trim()?.length !== 0 && validPath
+          : data.invokeNamespace !== undefined && data.invokeNamespace?.trim()?.length !== 0)
+      );
+    },
+    async performFinish(wizard: WebviewWizard, data: ParametersType): Promise<null> {
+      await vscode.window.withProgress(
+        {
+          cancellable: false,
+          location: vscode.ProgressLocation.Notification,
+          title: `Function Successfully invoke`,
+        },
+        async () => {
+          let invokeCommand: string;
+          if (data.invokeInstance === 'Local') {
+            invokeCommand = FuncAPI.invokeFunctionLocal(
+              data.invokeId,
+              data.invokePath,
+              data.invokeContextType,
+              data.invokeFormat,
+              data.invokeSource,
+              data.invokeType,
+              data.invokeDataMode === 'File' ? data.invokeDataFile : data.invokeDataText,
+            );
+          } else {
+            invokeCommand = FuncAPI.invokeFunctionRemote(
+              data.invokeId,
+              data.invokeNamespace,
+              data.invokeContextType,
+              data.invokeFormat,
+              data.invokeSource,
+              data.invokeType,
+              data.invokeDataMode === 'File' ? data.invokeDataFile : data.invokeDataText,
+            );
+          }
+          const result: CliExitData = await executeCmdCli.executeExec(invokeCommand);
+          if (result.error) {
+            // eslint-disable-next-line @typescript-eslint/no-floating-promises
+            vscode.window.showErrorMessage(`Fail invoke Function: ${getStderrString(result.error)}`);
+            return false;
+          }
+
+          // eslint-disable-next-line @typescript-eslint/no-floating-promises
+          vscode.window.showInformationMessage('Function Successfully invoke');
+          return true;
+        },
+      );
+      return null;
+    },
+    getNextPage(): IWizardPage | null {
+      return null;
+    },
+    getPreviousPage(): IWizardPage | null {
+      return null;
+    },
+  },
+};
+
+function invokeFunctionForm(context: vscode.ExtensionContext): WebviewWizard {
+  const data: Map<string, string> = new Map<string, string>();
+  const wiz: WebviewWizard = new WebviewWizard('invoke-function', 'invoke-function', context, def, data);
+  return wiz;
+}
+
+export function invokeFunction(context: vscode.ExtensionContext): void {
+  const wiz: WebviewWizard = invokeFunctionForm(context);
+  wiz.open();
+}
+
+export function createInvokeFunction(context: vscode.ExtensionContext): void {
+  const getEnvFuncId = !process.env.FUNC_ID ? 'ca8758fc-3bcc-4057-871e-5cea37fa215b' : process.env.FUNC_ID;
```

Comment:
Not sure how good is to always pre-fill this field with the same value. If you don't pass any ID to the func cli it generates an id casually. But here you show the field with a value so most users will not change it and all invoke will use the same ID. Then there is no way to know that if you empty the textbox the ID will be generated automatically.